docs/Safe Curve criteria

```diff
@@ -11,7 +11,7 @@ Safe Curve criteria checklist:
 - [x] The curve must admit a Montgomery ladder; this effectively restricts the shape to Montgomery or [twisted] Edwards 
 - [ ] The security against "combined attacks" on the twist, which is the a generalisation of rho security, must be > 2^100 
 - [x] The curve must admit a simple complete addition law; this further restricts which Montgomery and Edwards curves are admitted 
-- [x] The Elligator 2 algorithm for hashing to the curve must work 
+- [ ] The Elligator 2 algorithm for hashing to the curve must work 
```

Comment:
Any news about what do you need to close this?

Reply:
Yeah, solve it.